test/test_time_series.jl

```diff
@@ -211,6 +211,36 @@ end
     forecast_retrieved =
         IS.get_time_series(IS.Probabilistic, component, "test"; start_time = initial_time)
     @test IS.get_initial_timestamp(forecast_retrieved) == initial_time
+    t = IS.get_time_series_array(
+        IS.Probabilistic,
+        component,
+        "test";
+        start_time = initial_time,
+    )
+    @test size(t) == (24, 99)
```

Comment:
No data is validated in this test.  Each item is 1.0.  Maybe that's covered elsewhere.  Consider where the data needs to be unique and then verified after the read.